CONTRIBUTING.rst

```diff
@@ -237,7 +235,7 @@ Supported versions can be found in our ``noxfile.py`` `config`_.
 .. _config: https://github.com/googleapis/python-pubsublite/blob/main/noxfile.py
 
 
-We also explicitly decided to support Python 3 beginning with version 3.7.
+We also explicitly decided to support Python 3 beginning with version 3.8.
 Reasons for this include:
```

Comment:
I've opened https://github.com/googleapis/synthtool/pull/1766 to remove this section from CONTRIBUTING.rst as it is obsolete.